charts/gatekeeper/README.md

```diff
@@ -126,6 +126,7 @@ _See [Exempting Namespaces](https://open-policy-agent.github.io/gatekeeper/websi
 | validatingWebhookAnnotations                | The annotations to add to the ValidatingWebhookConfiguration                                                                                                                                                                                                        | `{}`                                                                  |
 | validatingWebhookObjectSelector               | The label selector to further refine which namespaced resources will be selected by the webhook. Please note that an exemption label means users can circumvent Gatekeeper's validation webhook unless measures are taken to control how exemption labels can be set.                                           | `{}`                                                                      |
 | validatingWebhookCheckIgnoreFailurePolicy     | The failurePolicy for the check-ignore-label validating webhook                                                                                                                                                                                     | `Fail`                                                                    |
+| validatingWebhookCustomNamespaceSelectors     | Additional namespace selectors | `{}`                                                                      |
```

Comment:
Can you pls revert all the changes made in `charts/`?
PTAL https://open-policy-agent.github.io/gatekeeper/website/docs/help#contributing-to-helm-chart